Automattic/woocommerce-payments
# Move the email field to the top of the Checkout page when StripeLink is enabled

Fixes #3801 

#### Changes proposed in this Pull Request
Moving the billing email field to the top of the Checkout page when StripeLink is enabled

#### Testing instructions
- Navigate to WooCommerce -> Settings -> Payments.
- Enable StripeLink gateway under Express checkouts.
- Add a Product to your Cart and navigate to the Checkout page.

Case 1:
- Verify the Email field is the first field under the Billing information section.

Case 2
- Open the checkout page with a saved email address.
- Verify the Link button is displayed in the email address field.

-------------------


**Post merge**
